### ISSUE TITLE
'Filter By' feature of team members by practice area

### ISSUE THREAD

**Issue by buddy-harlow** [Bixal/bixal-site-drupal#242]
## Location: https://www.bixal.com/about
- 'Filter By' menu is not keyboard accessible. (Cannot tab to select each area)
- Menu should also have a keyboard-accessible 'Close' button to dismiss the new content
- Once the menu is closed, the focus should be back on the 'Filter By' button.
- When you filter by a specific practice area, users need to be notified that they are applying a filter, the filter has successfully been applied, and the number of results.

## Suggestion
Remove the 'Filter By' functionality and menu completely. This is something we can iterate on later and make accessible, but this component is a critical failure.
